### PR TITLE
Respin 16.18.0, 18.11.0, 19.0.0 [release]

### PR DESCRIPTION
**Pull request by JalexChen**
The versions in the title in addition to 18.12.0 and 14.21.0

### CODE REVIEW

File: GEN-CHECK [hunk 1]
```diff
@@ -1 +1 @@
-GEN_CHECK=(18.12.0=lts)
+GEN_CHECK=(18.12.0 14.21.0)
```
Comment (felicianotech) [inline GEN-CHECK:1]: The lts tag was forgotten as well as the other versions this PR builds.

File: push-images.sh [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
-docker tag cimg/node:18.12.0 cimg/node:lts
-docker push cimg/node:lts
 docker push cimg/node:18.12-browsers
 docker push cimg/node:18.12.0-browsers
-docker tag cimg/node:18.12.0-browsers cimg/node:lts-browsers
-docker push cimg/node:lts-browsers
+docker push cimg/node:14.21
+docker push cimg/node:14.21.0
+docker push cimg/node:14.21-browsers
+docker push cimg/node:14.21.0-browsers
```
Comment (felicianotech) [inline push-images.sh:10]: Some versions getting built in this PR are missing here.

**Review by felicianotech — CHANGES_REQUESTED** [review]

File: push-images.sh (hunk 2 above)
Comment (JalexChen) [reply push-images.sh:10]: Fixing

File: GEN-CHECK (hunk 1 above)
Comment (JalexChen) [reply GEN-CHECK:1]: Ah. I didn't include lts because I thought it was already specified from a prior release in the ALIAS file. Fixing

**Review by felicianotech — APPROVED** [review]

**Pull request merged**